test/blocks/merch/merch.test.js

```diff
@@ -307,11 +307,12 @@ describe('Merch Block', () => {
 
   describe('Tacocat config', () => {
     it('falls back to en for unsupported languages', async () => {
-      const { literalScriptUrl, language } = getTacocatEnv('local', { ietf: 'xx-US' });
+      const { literalScriptUrl, language, tacocatEnv } = getTacocatEnv('local', { ietf: 'xx-US' });
       expect(literalScriptUrl).to.equal(
         'https://www.stage.adobe.com/special/tacocat/literals/en.js',
       );
       expect(language).to.equal('en');
+      expect(tacocatEnv).to.equal('PRODUCTION');
```

Comment:
if you define this as a const as rares said above, could import it into the test and use the var here to keep things dry.